charts/tidb-backup/templates/scripts/_start_backup.sh.tpl

```diff
@@ -19,3 +19,11 @@ uploader \
   --bucket={{ .Values.backup.gcp.bucket }} \
   --backup-dir=/data/${dirname}
 {{- end }}
+
+{{- if .Values.backup.ceph }}
+uploader \
```

Comment:
Do we need an updated uploader docker image?

Reply:
Yes. The pr just merged on https://github.com/tennix/tidb-cloud-backup.